src/main.cpp

```diff
@@ -67,6 +67,8 @@ bool fExperimentalMode = false;
 bool fImporting = false;
 bool fReindex = false;
 bool fTxIndex = false;
+bool fInsightExplorer = false;
+bool fAddressIndex = false;     // same value as fInsightExplorer
```

Comment:
I like the change to use `-indexexplorer` as a single flag. I think that we should either use boolean variable `fAddressIndex` or replace it with `fInsightExplorer`